docs/faq.rst

```diff
@@ -191,6 +191,11 @@ are `numpydoc <https://github.com/numpy/numpydoc>`_ and
 output more closely matches the format of standard Sphinx annotations,
 and as a result, it tends to look a bit better with the default theme.
 
+.. note::
+
+   To use these extensions you need to :doc:`specify the dependencies <guides/specifying-dependencies>`
+   using a requirements file.
```

Comment:
A dependency does not necessarily has to be in a requirements file, the guide mention all possible ways.

Reply:
Updated!